source/includes/rest/_orders.md

```diff
@@ -0,0 +1,842 @@
+              <td><code>taxable</code></td>
+              <td>Boolean indicating whether the item is taxable.</td>
+            </tr>
+            <tr>
+              <td><code>properties</code></td>
+              <td>An Object containing properties about the line item.</td>
+            </tr>
+          </tbody>
+        </table>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Create or update a refund
+
+> To create or update a refund for an order:
+
+```shell
+curl -X POST "https://api.getdrip.com/v2/YOUR_ACCOUNT_ID/ORDER_ID/refunds" \
+  -H 'User-Agent: Your App Name (www.yourapp.com)' \
+  -u YOUR_API_KEY: \
+  -d @- << EOF
+  {
+    "refunds": [{
+        "upstream_id": "tuvwx",
+        "amount": 2000,
+        "note": "Incorrect size",
+        "processed_at": "2013-06-22T10:41:11Z"
+      }]
+  }
+  EOF
+```
+
+
+> Responds with a `202 Accepted` and an empty JSON response:
+
+```json
+{}
+```
+
+When a refund is created, the subscriber's `lifetime_value` attribute
+will be automatically decremented by the total amount of the refund.
+
+### HTTP Endpoint
+
+`POST /:account_id/orders/ORDER_ID/refunds`
+
+### Arguments
+
+<table>
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>amount</code></td>
+      <td>Required. The amount of the refund.</td>
+    </tr>
+    <tr>
+      <td><code>upstream_id</code></td>
+      <td>Optional. The unique id of refund in the order management system.</td>
+    </tr>
+    <tr>
+      <td><code>note</code></td>
+      <td>Optional. A note about the refund.</td>
+    </tr>
+    <tr>
+      <td><code>processed_at</code></td>
+      <td>Optional. The String time at which the refund was processed in <a href="http://en.wikipedia.org/wiki/ISO_8601">ISO-8601</a> format.</td>
+    </tr>
+  </tbody>
+</table>
+
+## Order events
+
+Drip automatically records a number of events in the life cycle of an order. You can find
+triggers for these events by selecting <code>Drip Order API</code> from the provider dropdown in your Drip automation.
+
+<table>
+  <thead>
+    <tr>
+      <th>Event</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>Placed an order</code></td>
+      <td>This event is fired when an order is created.</td>
+    </tr>
+    <tr>
+      <td><code>Updated an order</code></td>
+      <td>This event is fired when an existing order is updated.</td>
+    </tr>
+    <tr>
+      <td><code>Paid an order</code></td>
+      <td>This event is fired when the <code>financial_state</code> on an order is set to <code>paid</code>.</td>
+    </tr>
+    <tr>
+      <td><code>Fulfilled an order</code></td>
+      <td>This event is fired when the <code>fulfillment_state</code> on an order is set to <code>fulfilled</code>.</td>
+    </tr>
+    <tr>
+      <td><code>Canceled an order</code></td>
+      <td>This event is fired when the <code>cancelled_at</code> time on an order is set.</td>
```

Comment:
@iantance Will differences in the column name, `cancelled_at`, and the actual "Canceled an order" event create some confusion since they're spelled differently? Feels like we should maintain the same spelling all around. I've certainly had situations where stuff didn't work because of oversight with stuff like this.

Reply:
I am a little concerned about that, too. We have an unfortunate mix of `canceled` and `cancelled` throughout the app (e.g. Subscriptions get `cancelled` while Broadcasts get `canceled`).

I think the data model/column on Purchases uses `cancelled_at`, but the spec for the events used `Canceled` and I followed along. Changing the column is more of a hassle, but I admit that I have a slight preference for the single `l`.

@bfortune347 @tayjohno Do either of you have feeling about this?

Reply:
Yeah, when I've looked this up in the past, the single "l" is more common in the US, where double "l" wins in the UK. I'd rather keep the presentation layer with a single "l", though do agree that it's confusing to have the database naming differ. Ian, could we rename this to a single "l" as part of the Purchases -> Orders database migration issue?

Reply:
A sort-of-jingoistic mnemonic for this is to remember that the UK took the "l" in 1776 🇺🇸 🎆.

I'm not certain on the procedure for that migration, so I'm not sure if it would be easier to rename a column as part of the table rename or separately.

Reply:
Nice mnemonic :). 

@tayjohno, and thoughts on what's easier in terms of a potential column rename?

Reply:
I think the DB transfer would be the perfect time to rename. I agree with @bfortune347, we should go for the single-'L' as much as possible. I think the issue is that we have some incoming services that use double-`L` for their events, and then we let that spelling propagate through our system.

Reply:
OK, updated that issue so we can handle it there. 

@iantance, anything else we're waiting for before we launch this one?